use std::error::Error;

mod console;
mod core;

fn main() -> Result<(), Box<dyn Error>> {
    Ok(())
}
